seminar_registration_system/member_info_capture/models.py

```diff
@@ -12,7 +84,68 @@ class Member(models.Model):
         choices=STUDENT_TYPE_CHOICES,
         default=NEW_STUDENT,
     )
+
     last_name = models.CharField(max_length=50)
     first_name = models.CharField(max_length=50)
-    personal_email = models.EmailField(max_length=254)
-    residential_address = models.CharField(max_length=200)
+    personal_email = models.EmailField(max_length=254, validators=[validate_email])
+    residential_address = models.CharField(max_length=200 )
+    system_email =  models.EmailField(max_length=254, null=False, unique=True)
```

Comment:
the plan is to make the user have an automated system email if it does not exist for example if Last name = Monc and First names = Mfundo Thuto, what email does it takes the last name and initials of the first names assign random number perhaps starting from 01, this helps with same identities
```suggestion
    system_email =  models.EmailField(max_length=254, null=False, unique=True)
```